### PR TITLE
Tests: Add Sketcher tests to CI

### PR DESCRIPTION
**Pull request by chennes**
Now that Sketcher has independent C++ tests, it needs to be explicitly added to the CI.

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9368">pipeline status</a> for feature branch [PR_9368](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9368). Pipeline [845803531](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/845803531) was triggered at [4a1bc4f](https://github.com/FreeCAD/FreeCAD/pull/9368/commits/4a1bc4f1c7b66313b94bd1f451e01c5751d72ad6). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9368">pipeline status</a> for feature branch [PR_9368](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9368). Pipeline [845890719](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/845890719) was triggered at [127be50](https://github.com/FreeCAD/FreeCAD/pull/9368/commits/127be504ea6c9eca6b3a402ce3167c55465a4e03). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9368">pipeline status</a> for feature branch [PR_9368](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9368). Pipeline [845895546](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/845895546) was triggered at [79498dc](https://github.com/FreeCAD/FreeCAD/pull/9368/commits/79498dcd086a17dd752af95a6062f93936aaa26f). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by 0penBrain** [comment]
@chennes just to let you know I probably won't be able to review before Wednesday. 😉

File: .github/workflows/actions/runCPPTests/runAllTests/action.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# *   <https://www.gnu.org/licenses/>.                                      *
+# *                                                                         *
+# ***************************************************************************
+
+name: Run all C++ Tests
+description: "Run all C++ tests, generating logs and reports for each"
+
+runs:
+  using: "composite"
+  steps:
```
[the remark this answers is not in the document]
Comment (chennes) [reply .github/workflows/actions/runCPPTests/runAllTests/action.yml:40]: Agreed, all of our test output is very difficult to parse right now. I don't know how to do what you are describing though. Can we work on it together in a follow-on PR?

[the remark this answers is not in the document]
Comment (0penBrain) [reply .github/workflows/actions/runCPPTests/runAllTests/action.yml:40]: > Can we work on it together in a follow-on PR?

Sure. Or if there is no matter of urgency, we can do it in this branch in coming days. Up to you. May you squash the commits ?

**Comment by 0penBrain** [comment]
OK I started doing some trials. I"m testing on my fork. Expect a bit of delay as I had no more CI caches available. :)

**Comment by 0penBrain** [comment]
@chennes Just pushed a PR to your branch. Let you check and see. ;)

**Comment by chennes** [comment]
> @chennes Just pushed a PR to your branch. Let you check and see. ;)

Looks great, thanks!

**Comment by 0penBrain** [comment]
Looks good to me (but I'm obviously no more neutral when reviewing 😈 )

**Pull request merged**